### ISSUE TITLE
Feature Request: Add support for Webhook integration

### ISSUE THREAD

**Issue by bkimbrough-bw** [terraform-providers/terraform-provider-datadog#344]
This feature request is for a datadog_integration_webhook resource. It would be useful to configure the Webhook Datadog integration in Terraform code.

Datadog docs: https://docs.datadoghq.com/api/?lang=bash#integration-webhooks

Similar Issues: #67 #75 #76  #196

**Comment by jmcarp** [comment]
Is anybody working on this right now? cc @bedwards

**Comment by bkimbrough88** [comment]
> Is anybody working on this right now? cc @bedwards

I had been waiting on https://github.com/zorkian/go-datadog-api/pull/265, but just now saw that it had been merged for some time now. You're welcome to take it, otherwise I will look to create a PR this weekend.

**Comment by juanitomint** [comment]
+1 we need this too
@wildlifestudios.com

**Comment by glasseyes42** [comment]
Any news on whether this will be looked into? What with https://github.com/zorkian/go-datadog-api/pull/300 having been merged in place of 265. Could really use this in the official provider, I can't say I'm a fan of my hack whereby I've wrapped the relevant api path with https://github.com/Mastercard/terraform-provider-restapi

**Comment by bkimbrough88** [comment]
> Any news on whether this will be looked into? What with [zorkian/go-datadog-api#300](https://github.com/zorkian/go-datadog-api/pull/300) having been merged in place of 265. Could really use this in the official provider, I can't say I'm a fan of my hack whereby I've wrapped the relevant api path with https://github.com/Mastercard/terraform-provider-restapi

I opened #413 2 weeks after that was merged and it was essentially rejected by DataDog as they wanted to wait for their new API implementation, which is now a year in the making. Maybe @bkabrda has more details, but this is the last I heard on the matter:

`
Hey 👋 the DD web integrations team is working on a new set of API endpoints that will allow manipulating individual webhooks. I'd like to postpone reviewing/merging this PR until after this work is done, so that we could take a similar approach that we took with the pagerduty integration (where we have the top-level pagerduty integration object and then resource for individual pagerduty service objects).
`

**Comment by therve** [comment]
Hi,

The new API has been generated in go: https://github.com/DataDog/datadog-api-client-go/pull/1071. We should be able to add the new resources soon. Thanks for your patience.
